src/NewsletterEpic/index.tsx

```diff
@@ -11,26 +11,10 @@ import {
     headline,
     textSans,
 } from '@guardian/source-foundations';
-import { Button, buttonThemeBrandAlt } from '@guardian/source-react-components';
+import { Button, buttonThemeBrandAlt, SvgClock } from '@guardian/source-react-components';
 import { COMPONENT_NAME, canRender } from './canRender';
 import { LoadingDots } from './LoadingDots';
 
-// Once https://github.com/guardian/source/pull/843 is merged and in a
-// @guardian/src-icons release we'll be able to bump the version on this project
-// and replace this with:
-// import { SvgClock } from '@guardian/src-icons'
```

Comment:
This is now available in `@guardian/source-react-components` as source was restructured in the meantime.